### DISCUSSION TITLE
请问大佬们，如何优化偶尔慢的问题！

### DISCUSSION THREAD

**Discussion by zyw8136** [apache/doris#8327] [A - General / Q&A]
我有一个sql在一直执行，但是总出现这样的查询慢，这个如何优化，从哪里入手，还请高手们指点一下！
![image](https://user-images.githubusercontent.com/4756695/156679255-cfa82a05-dbfe-4d9e-9d7c-68fcbbcf499d.png)

**Comment by dataalive** [comment]
you could try to get metrics during that time as follows : 

- CPU / IO Util
- BE log(compaction, error or sth like this) 
- FE GC log
